Source/buildbindinggo.go

```diff
@@ -433,12 +437,12 @@ func buildGoClass(component ComponentDefinition, class ComponentDefinitionClass,
 		w.Writeln("")
 		w.Writeln("// Equal reports whether inst and w refer to the same C pointer.")
 		w.Writeln("func (inst %s) Equal(w %s) bool {", class.ClassName, class.ClassName)
-		w.Writeln("  return inst.ref == w.ref")
+		w.Writeln("  return inst.Ref == w.Ref")
 		w.Writeln("}")
 	} else {
-		w.Writeln("func (wrapper * Wrapper) new%s(r ref) %s {", class.ClassName, class.ClassName)
```

Comment:
@alexanderoster this `new` methods are internal helper functions to be called when instantiating objects inside public methods, but I don't think they should be made public as they do not provide value by their own.